packages/edit-post/src/components/layout/index.js

```diff
@@ -12,11 +12,11 @@ import {
 	UnsavedChangesWarning,
 	EditorNotices,
 	PostPublishPanel,
+	EditorKeyboardShortcutsRegister,
```

Comment:
Why do we have inconsistency that from the `block-editor` package, the component which manages shortcut registration is defined as `BlockEditorKeyboardShortcuts.Register` (a property of `BlockEditorKeyboardShortcuts`), but then the same sort of component from `editor` package has its own standalone exported member `EditorKeyboardShortcutsRegister`?

Reply:
because the Editor one registers both the Text and Visual shortcuts that can't be applied at the same time, another option is to split the registration into two components as well but I feel a better pattern is that each package would have a single register component.